build_scripts/windows/azure-cli.wixproj

```diff
@@ -21,6 +12,15 @@
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' AND '$(MSBuildExtensionsPath32)' != '' ">$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' ">$(MSBuildExtensionsPath)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
   </PropertyGroup>
+  <!-- Local WiX -->
+  <PropertyGroup>
+    <LocalWixRoot>artifacts\wix</LocalWixRoot>
+    <WixToolPath>$(LocalWixRoot)</WixToolPath>
+    <WixTargetsPath Condition="Exists('$(WixToolPath)\Wix.targets')">$(WixToolPath)\Wix.targets</WixTargetsPath>
+    <WixTasksPath Condition="Exists('$(WixToolPath)\wixtasks.dll')">wixtasks.dll</WixTasksPath>
```

Comment:
I was wondering why we need to add `Condition="Exists('$(WixToolPath)\Wix.targets')"` here.

Reply:
It's to support building locally via msbuild instead of your scripts that do more. For quick testing, I just run msbuild from a VS Dev prompt. It won't hurt anything outside of that but will be easier for future edits.